src/lp_sensitivity.jl

```diff
@@ -0,0 +1,322 @@
+
+    cur_row_idx = 0
+
+    for (F, S) in con_types
+        constraints = all_constraints(model, F, S)
+        if F <: VariableRef
+            for constraint in constraints
+                con_obj = constraint_object(constraint)
+                j = var_to_idx[con_obj.func]
+                range = MOI.Interval(con_obj.set)
+                var_lower[j] = max(var_lower[j], range.lower)
+                var_upper[j] = min(var_upper[j], range.upper)
+            end
+            #issue 1892 makes is hard to know how to handle variable constraints, atm. assume unique variable constraints.
+            continue
+        end
+        for constraint in constraints
+            cur_row_idx += 1
+            con_obj = constraint_object(constraint)
+            con_terms = con_obj.func.terms
+            push!(affine_constraints, constraint)
+            for (var, coeff) in con_terms
+                push!(col_j, var_to_idx[var])
+                push!(row_i, cur_row_idx)
+                push!(coeffs, coeff)
+            end
+            push!(col_j, cur_row_idx + length(vars))
+            push!(row_i, cur_row_idx)
+            push!(coeffs, -1.0)
+
+            range = MOI.Interval(con_obj.set)
+            push!(var_lower, range.lower)
+            push!(var_upper, range.upper)
+        end
+    end
+
+    A = sparse(row_i, col_j, coeffs, cur_row_idx, cur_row_idx + length(vars))
+    return A, var_lower, var_upper, affine_constraints
+end
+
+"""
+Returns the optimal primal values for the problem in standard form, c.f. `_std_matrix(model)`.
+"""
+function _std_primal_solution(model::Model, constraints::Vector{ConstraintRef})
+    vars = all_variables(model)
+    x = [value.(vars); value.(constraints)]
+    return x
+end
+
+"""
+Returns the basis status of all variables of the problem in standard form, c.f. `_std_matrix(model)`.
+"""
+function _std_basis_status(model::Model)::Vector{MOI.BasisStatusCode}
+    con_types = list_of_constraint_types(model)
+    vars = all_variables(model)
+    var_to_idx = Dict{VariableRef, Int64}()
+    for (j, var) in enumerate(vars)
+        var_to_idx[var] = j
+    end
+    basis_status = fill(MOI.BASIC, length(vars))
+
+    for (F, S) in con_types
+        constraints = all_constraints(model, F, S)
+        if F <: VariableRef
+            for constraint in constraints
+                con_obj = constraint_object(constraint)
+                j = var_to_idx[con_obj.func]
+                basis_status[j] = MOI.get(model, MOI.ConstraintBasisStatus(), constraint)
+                if S <: MOI.LessThan && basis_status[j] == MOI.NONBASIC
+                    basis_status[j] = MOI.NONBASIC_AT_UPPER
+                elseif S <: MOI.GreaterThan && basis_status[j] == MOI.NONBASIC
+                    basis_status[j] = MOI.NONBASIC_AT_LOWER
+                end
+            end
+            #issue 1892 makes is hard to know how to handle variable constraints, atm. assume unique variable constraints.
+            continue
```

Comment:
Instead of this continue,
```julia
if F <: VariableRef
    # ... what you have ...
else
    for constraint in constraints
        # ...
    end
end
```